pkg/vcdsdk/gateway.go

```diff
@@ -1614,10 +1615,21 @@ func (gm *GatewayManager) CreateLoadBalancer(ctx context.Context, virtualService
 	}
 
 	if externalIP == "" {
-		externalIP, err = gm.GetUnusedExternalIPAddress(ctx, gm.IPAMSubnet)
+		isGatewayUsingIpSpaces, err := gm.IsUsingIpSpaces()
 		if err != nil {
-			return "", fmt.Errorf("unable to get unused IP address from subnet [%s]: [%v]",
-				gm.IPAMSubnet, err)
+			return "", fmt.Errorf("unable to create load balancer. err [%v]", err)
+		}
+		if isGatewayUsingIpSpaces {
```

Comment:
Can we add a log-info statement stating the type of the gateway used (using IP spaces or not)